db/seeds.rb

```diff
@@ -1,7 +1,18 @@
-FactoryGirl.create(:user)
+20.times do
+  FactoryGirl.create(:user,
+    first_name: Faker::Name.first_name,
+    last_name:  Faker::Name.last_name,
+    email:      Faker::Internet.email)
+end
 
```

Comment:
Bad news, kids--no using Factory Girl for seed data. Create some objects manually pls. Why [here](https://robots.thoughtbot.com/factory_girl-for-seed-data).